### ISSUE TITLE
Here is the return from BTC/USDT :

### ISSUE THREAD

**Issue by MRCTRADE** [ccxt/ccxt#14776]
Here is the return from BTC/USDT : 
```
[1660780800000.0, 23335.92, 23594.7, 23097.68, 23187.3, 1689.865001]
[1660867200000.0, 23187.3, 23199.37, 20785.07, 20827.96, 5178.872814]
[1660953600000.0, 20827.96, 21370.11, 20768.12, 21137.82, 2911.976528]
[1661040000000.0, 21137.82, 21783.45, 21068.22, 21516.52, 1931.580462]
[1661126400000.0, 21516.52, 21540.0, 20899.33, 21325.05, 1966.100943]
```

And the value from exchange itself https://www.bybit.com/en-US/trade/spot/BTC/USDT : 
Timestamp 1660780800000 (18.08.2022): 
<img width="581" alt="TS 1660780800000" src="https://user-images.githubusercontent.com/53877621/185978175-4a04f9cd-469f-4bf7-b66e-5c449422712b.png">
Timestamp 1660867200000 (19.08.2022): 
<img width="578" alt="1660867200000" src="https://user-images.githubusercontent.com/53877621/185978239-546a25b9-61ff-458d-8b4e-b305884f688c.png">
And Timestamp 1661040000000 (21.08.2022):
<img width="629" alt="1661040000000" src="https://user-images.githubusercontent.com/53877621/185978483-e599a6ab-c721-4155-a940-2c84161eb0e6.png">

So basically it works and give the exact same information.

Please note that by default bybit on CCXT is configured in 'swap'.
If you want to have spot you need to override it by using : 
`exchange = ccxt.bybit({'options': {'defaultType': 'spot'}})`
Also, on the exchange page make sure you are looking at the spot trading of BTC/USDT and not the price of the derivatives USDT perpetual.

_Originally posted by @pier0074 in https://github.com/ccxt/ccxt/issues/14771#issuecomment-1222655890_

**Comment by MRCTRADE** [comment]
thank you so much for the very complete and detailed answer, I see that the data match but mine do not
for what reason?

import ccxt
#exchange = ccxt.bybit()
exchange = ccxt.bybit({'options': {'defaultType': 'spot'}})

ohlc = exchange.fetch_ohlcv('BTC/USDT',timeframe='1d',limit=5)
for candle in ohlc:
    print(candle)

Here is the return from BTC/USDT :
[1660867200000, 23188.0, 23198.0, 20754.5, 20819.0, 312308.071]
[1660953600000, 20819.0, 21365.0, 20730.0, 21124.0, 153410.543]
[1661040000000, 21124.0, 21780.0, 21060.5, 21505.5, 134458.04]
[1661126400000, 21505.5, 21538.0, 20858.0, 21382.5, 182711.163]
[1661212800000, 21382.5, 21455.0, 20885.0, 21232.5, 48179.721]

Timestamp 1660867200000 (Friday, 19 August 2022 00:00:00)
<img width="1440" alt="Screenshot 2022-08-23 at 10 57 27" src="https://user-images.githubusercontent.com/49906349/186116983-c01ef321-3259-453b-aa32-a0a3108f905a.png">

Timestamp 1660953600000 (Saturday, 20 August 2022 00:00:00)
<img width="1440" alt="Screenshot 2022-08-23 at 11 14 18" src="https://user-images.githubusercontent.com/49906349/186120631-1710dbc6-bc66-4e6e-be79-49c4c7228e07.png">

Timestamp 1661040000000 (Sunday, 21 August 2022 00:00:00)
<img width="1440" alt="Screenshot 2022-08-23 at 11 22 42" src="https://user-images.githubusercontent.com/49906349/186122474-1f1094fc-f329-424e-965e-b5119bc9dda9.png">

Timestamp 1661126400000 (Monday, 22 August 2022 00:00:00)
<img width="1440" alt="Screenshot 2022-08-23 at 11 26 08" src="https://user-images.githubusercontent.com/49906349/186123144-13ba81fe-168d-43a8-b69a-e9843c388b7b.png">

**Comment by pier0074** [comment]
As I cannot reproduce this difference it will be hard for me to help you.
Could you please indicate the version of ccxt you are using, the version of python and your OS?

**Comment by MRCTRADE** [comment]
> As I cannot reproduce this difference it will be hard for me to help you.
> Could you please indicate the version of ccxt you are using, the version of python and your OS?

I really appreciate your help
CCXT Version: 1.72.89
Python 3.8.8
ProductName:	macOS
ProductVersion:	12.5.1
BuildVersion:	21G83

**Comment by pier0074** [comment]
When I downgrade to your version I obtain the same wrong result as you.
Please upgrade to the newest version and the problem you be resolved.

**Comment by MRCTRADE** [comment]
thanks for the suggestion I have upgraded and the problem has been solved for the spot market
now with this code i tried to do the same thing for the market Derivates USDT Perpetual on symbol BTCUSDT
The candlestick data in this case does not match

import ccxt
#exchange = ccxt.bybit()
exchange = ccxt.bybit({'options': {'defaultType': 'future'}})
#exchange = ccxt.bybit({ 'option': { 'defaultMarket': 'futures' } })
ohlc = exchange.fetch_ohlcv('BTCUSDT',timeframe='1d',limit=5)
for candle in ohlc:
    print(candle)

**Comment by pier0074** [comment]
Working fine with me.
```
[1660953600000, 20819.0, 21365.0, 20730.0, 21124.0, 153410.543]
[1661040000000, 21124.0, 21780.0, 21060.5, 21505.5, 134458.04]
[1661126400000, 21505.5, 21538.0, 20858.0, 21382.5, 182711.163]
[1661212800000, 21382.5, 21678.5, 20885.0, 21516.5, 169079.107]
[1661299200000, 21516.5, 21537.5, 21138.5, 21275.0, 59082.258]
```
see : https://www.bybit.com/trade/usdt/BTCUSDT

Please send your output.

**Comment by MRCTRADE** [comment]
[1660953600000, 20819.0, 21365.0, 20730.0, 21124.0, 153410.543]
[1661040000000, 21124.0, 21780.0, 21060.5, 21505.5, 134458.04]
[1661126400000, 21505.5, 21538.0, 20858.0, 21382.5, 182711.163]
[1661212800000, 21382.5, 21678.5, 20885.0, 21516.5, 169079.107]
[1661299200000, 21516.5, 21537.5, 21138.5, 21288.0, 59473.945]
I apologize I had set the candles to Heikin-Ashi and clearly could not be correct, after setting the candles to Bars the values match

**Comment by samgermain** [comment]
> [1660953600000, 20819.0, 21365.0, 20730.0, 21124.0, 153410.543] [1661040000000, 21124.0, 21780.0, 21060.5, 21505.5, 134458.04] [1661126400000, 21505.5, 21538.0, 20858.0, 21382.5, 182711.163] [1661212800000, 21382.5, 21678.5, 20885.0, 21516.5, 169079.107] [1661299200000, 21516.5, 21537.5, 21138.5, 21288.0, 59473.945] I apologize I had set the candles to Heikin-Ashi and clearly could not be correct, after setting the candles to Bars the values match

Sounds good, I'm going to close this issue. Feel free to reopen it if you have any more questions
